tests/incremental_mode/run.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+TASK_NAME="test"
+
+function run() {
+    run_sql_file $cur/data/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1
+    check_contains 'Query OK, 2 rows affected'
+    run_sql_file $cur/data/db2.prepare.sql $MYSQL_HOST2 $MYSQL_PORT2
+    check_contains 'Query OK, 3 rows affected'
+
+    run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+    run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
+    run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
+
+    check_port_alive $MASTER_PORT
+    check_port_alive $WORKER1_PORT
+    check_port_alive $WORKER2_PORT
+
+    cd $cur && GO111MODULE=on go build -o bin/dmctl dmctl.go && cd -
+    cd $cur && GO111MODULE=on go build -o bin/dmctl_stop dmctl_stop.go && cd -
```

Comment:
can we re-use these binaries between multiple test cases?

Reply:
added global dmctl tools, now each test case can use them without re-compiling

Reply:
do we need to add retry for _dm_start_task.go_?

Reply:
currently not needed, If we need retry mechanism we can wrap the binary from caller